scss/gds.scss

```diff
@@ -0,0 +1,3 @@
+$button: true !default;
+
+@forward '../src/components/gds-button/gds-button' with ($output: $button);
```

Comment:
I opted for separate entry points depending on if you're looking for just the mixins or also the base styles. I bet down the line performance will suffer if we include everything. Also I kind of prefer separating the mixins in their own file as it makes the stencil sass file like a documentation on how to use the mixins.

Reply:
[Forward](https://sass-lang.com/documentation/at-rules/forward) seems like nice approach

Reply:
I have to learn this advanced sass syntax... 😬

Reply:
So is this what the `$output` is used for? For optimization?

Reply:
Yeah, but we could just skip it since you can anyways import individual components only and/or use purgecss.